app/Models/EntryDAOSQLite.php

```diff
@@ -28,6 +28,27 @@ public static function sqlIgnoreConflict(string $sql): string {
 		return str_replace('INSERT INTO ', 'INSERT OR IGNORE INTO ', $sql);
 	}
 
+	#[\Override]
+	protected static function sqlRegex(string $expression, string $regex, array &$values): string {
+		$values[] = $regex;
+		return "{$expression} REGEXP ?";
+	}
+
+	#[\Override]
+	protected function registerSqlFunctions(string $sql): void {
+		if (!str_contains($sql, ' REGEXP ')) {
+			return;
+		}
+		// https://php.net/pdo.sqlitecreatefunction
+		// https://www.sqlite.org/lang_expr.html#the_like_glob_regexp_match_and_extract_operators
+		$this->pdo->sqliteCreateFunction('regexp',
+			function (string $pattern, string $text): bool {
```

Comment:
It appears SQLite support isn't implemented yet. Just adding this comment so it's obvious when it is.

Reply:
Hum, SQLite is supposed to work already

Reply:
Sorry, that was from code inspection. I was looking for this block to do matching against the `im` like the others. I haven't migrated my database to check it yet though.  
This was just a comment to mark what was and wasn't included based on inspecting the code changes.